Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -941,6 +941,138 @@ public class VisualRecognition {
     }
  
     /**
+     Delete an image's metadata from a collection.
+     
+     - parameter withID: The ID of the collection to delete the image from.
+     - parameter imageID: The ID of the image to delete.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed when the image metadata is deleted successfully.
+    */
+    public func deleteImageMetadataFromCollection(
```

Comment:
Ahh I think this looks good 👍 Basically the same thing, but I feel that it might be clearer.